juan-pablo-lopez/Tarea 1/index.html

```diff
@@ -14,51 +14,52 @@
     <title>Cover Magazine</title>
 </head>
 <body>
-    <div class="grid-container">
-        <div class="grid-tittle">
+    <main class="grid-container">
+        <header class="grid-tittle">
             <h1>NOISE</h1>
-        </div>
-        <div class="grid-deck" >
+        </header>  
+        <section class="grid-deck" >
             <h2>FASHION AND TRAVEL MAGAZINE</h2>
-        </div>
-        <div class="grid-article" >
-            <div> 
+        </section>
+        <article class="grid-article" >
+            <section> 
                 <p>OVID</p> 
```

Comment:
@juanpablo-lopez01 , the content for all this <p> does not add semantic Value, try to put all the text inside one <p> the play with the width and text-wrap properties in CSS